Level 3 whitebox

jojo-ucf opened the pull request:
Creation of the Level 3 haunted house Whitebox.

jojo-ucf committed:
person scene change

sherbracewell committed:
Assisted in adding to Jody's scene

- Modified Life Pickup script
- Added canvas waypoints and life pickup  to scene

jojo-ucf committed:
HH Scene Update

-Create HH Floor Prefab for future use
-reworked camera (Still not working)

jojo-ucf committed:
ProBuilder Addition

I added Probuilder to the project

TriOmega commented:
Corrections: 
- "M_LightBlackMaterial 1" should be "M_LightBlack_01" (01 needed and use of word material is redundant)
- "JodyCameraTest" folder can be "Jody Camera Test"
- New Terrain file should be put somewhere and not left in Assets directory

Simple enough changes that I'll just go ahead and make them but keep in mind for next time.

TriOmega committed:
Update with tidying up for merge

- M_LightBlackMaterial 1 changed M_LightBlack_01
- JodyCameraTest folder changed to Jody Camera Test
New Terrain file moved to ProBuilder folder

TriOmega committed:
Merge branch 'main' into level-3-whitebox
